src/index.ts

```diff
@@ -1,49 +1,68 @@
-#!/usr/bin/env node --es-module-specifier-resolution=node
+#!/usr/bin/env node --no-warnings 
 
 import chalk from 'chalk'
-import figlet from 'figlet'
 import { Command } from 'commander'
 
+import packageJson from '../package.json' assert { type: 'json' }
 import { loadProcesses } from './lib/process/index.js'
 
+const { log } = console
+
+const version = packageJson.version
 const program = new Command()
 const example: string = JSON.stringify([{
   name: 'A test',
-  version: 1,
+  version: 2,
   program: [    
     { restriction: { SenderOwnsAllInputs: {} }} ,
+    { restriction: { None: {} }},
     { op: 'or' },
-    { restriction: { None: {} }} 
   ],
+},  {
+  name: 'B test',
+version: 2,
+program: [    
+  { restriction: { SenderOwnsAllInputs: {} }} ,
+  { restriction: { None: {} }}, 
+  { op: 'or' },
+],
 }])
  
 program
   .name('process management')
   .description('a command line interface for managing chain processes')
-  .version('1.4.0')
-
+  .version(version, '-v, --version', 'output current version')
 
 program.command('create')
   .description('A command for persisting process flows onto the chain')
-  .option('-d, --dryRun <bool>', 'performs a dry run', false)
-  .option('-h, --host <string>', 'substrate blockchain host address or FQDM, default - "localhost"', 'localhost')
-  .option('-p, --port <string>', 'specify host port number if it is not a default, default - 9944', '9944')
-  .option('-u, --user <string>', 'specify substrate blockhain user URI, default - "//Alice"', '//Alice')
+  .option('--dryRun', 'will not create a transaction and will run just against local chain')
```

Comment:
I'm not sure "will run just against local chain" either makes sense or is true.

Reply:
yea... it's not very clear let me update.

Reply:
It has been changed to

```js
  .option('--dryRun', 'to validate process and response locally before persisting on the chain, default - false')
```